full/cargo_global_config.toml

```diff
@@ -0,0 +1,8 @@
+[unstable]
```

Comment:
Can't we echo this in a file directly from the Dockerfile ? It would allow us not to have another file in the "full" directory (we only have dockerfiles so far in the directories). It makes the image file self contained... :man_shrugging: 

Don't know if it's better, what do you think ?

Reply:
@xchapron-ledger what do you think ?

Reply:
Yes, echo does the job as well, but decreases readability IMO:

 ```
 RUN echo "[unstable]" > $CARGO_HOME/config.toml
# Prevent bindgen tool to fail on Alpine (with '[host]' section)
# see https://ledger.slack.com/archives/C04SY25SREX/p1698055418107969
RUN echo "host-config = true" >> $CARGO_HOME/config.toml \
    && echo "target-applies-to-host = true" >> $CARGO_HOME/config.toml \
    && echo "[host]" >> $CARGO_HOME/config.toml \
    && echo "rustflags = [\"-Ctarget-feature=-crt-static\"]" >> $CARGO_HOME/config.toml
```

Reply:
Can also be done with something like this:
```
RUN printf '\
[unstable]\n\
\n\
# Prevent bindgen tool to fail on Alpine (with '[host]' section)\n\
# see https://ledger.slack.com/archives/C04SY25SREX/p1698055418107969\n\
host-config = true\n\
target-applies-to-host = true\n\
\n\
[host]\n\
rustflags = ["-Ctarget-feature=-crt-static"]\n\
' > config.toml
```
See https://stackoverflow.com/questions/33439230/how-to-write-commands-with-multiple-lines-in-dockerfile-while-preserving-the-new, but this is not really lisible

Reply:
If there is no better way to format it in the dockerfile, I would rather have it in another clean file as @yogh333 did it.

Reply:
Ok so let's keep a separated file for the moment ;)